source/playbooks/refining-and-improving.rst

```diff
@@ -32,6 +32,17 @@ Use the run timeline to help write an accurate retrospective. Events such as own
 .. image:: ../images/Retro.gif
    :alt: Create and publish retrospective reports.
 
+Metrics
+~~~~~~~
+
+Use metrics to identify key areas in which you want to measure performance and improvement, and extract valuable insights. 
+
+These metrics can be anything that's of interest to you and your team. For example, for a software release playbook you might want to have a metric tracking how many bugs were detected during a run. The output of the metrics you've added is provided in the retrospective and, over time, you can use the data to examine anomalies and refine goals.
+
+Another example is a support incident playbook. A metric such as time to resolution can be applied and used to identify areas that need more refinement such as tasks that might work better if they're split up so the goal is reached faster.
+
+Metrics are enabled when you enable retrospectives. Calibrate the type of metric you want to measure, and once a run is finished you can view the output in the retrospective. You can have multiple metrics configured per playbook and you can edit them at any time.
```

Comment:
"Metrics are enabled when you enable retrospectives." -- I feel that this may be important information to include at the beginning of this subsection because it's communicating a dependency between retros & metrics that the user may otherwise be unaware of.

Reply:
In addition, if this content was moved to the beginning of this section, you'd be able to remove the redundant text above: "The output of the metrics you've added is provided in the retrospective"